src/js/components/FilterTree.tsx

```diff
@@ -16,22 +15,80 @@ import Search from "../state/Search"
 import usePopupMenu from "./hooks/usePopupMenu"
 import {remote} from "electron"
 import Last from "../state/Last"
+import {SearchRecord} from "../types"
+
+const getPins = (node?: InvestigationNode): string[] => {
+  const result = reduce(
+    node?.getPath(),
+    (res, n) => {
+      res.push(n.model.filter)
+      return res
+    },
+    []
+  )
```

Comment:
I think might be simpler as a map function? `node.getPath().map(n => n.model.filter)`